### DISCUSSION TITLE
Updating the base docker os for PyPi

### DISCUSSION THREAD

**Discussion by Togtja** [pypiserver/pypiserver#614] [Q&A] [answered]
Hi, I saw that last week Pypi updated the docker image to Alpine 3.20 and python 3.10 (the [Commit](https://github.com/pypiserver/pypiserver/commit/0430375272fbc7488e4d2633befd5c156bdaffda))

I was wondering on how to use that.
While I assume it will be put onto the Unstable docker tag/branch. It would be really nice to run it on the stable 2.2.0 with the security vulnerabilities fixed on the docker container with 3.20. Any idea on how to best do this without crafting a docker image from scratch?

**Comment by dee-me-tree-or-love** [comment, marked as answer]
Hello @Togtja, thank you for raising this question 👍 

> While I assume it will be put onto the Unstable docker tag/branch.

That's indeed the case! Right now, the latest successful untagged commit on the `main` branch gets built and published as `pypiserver/pypiserver:unstable`. So if you would like, you could start using it directly. 

Of course I understand that in some contexts using `unstable` is not an option, so I am aiming to make a small release in the near future. I think it will be a path release `2.2.1` unless I'll include any new functionality. I think I'll make it before Christmas.
